src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export default function ChatbotLoadError(_props) {
```

Comment:
Do you need the parameter `_props` here?

Reply:
just habit, the short answer is no, I can remove